packages/editor/src/components/post-publish-panel/maybe-tags-panel.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+import { compose, ifCondition } from '@wordpress/compose';
+import { withSelect } from '@wordpress/data';
+import { PanelBody } from '@wordpress/components';
+
+import FlatTermSelector from '../post-taxonomies/flat-term-selector';
+
+const TagsPanel = () => {
+	const panelBodyTitle = [
+		__( 'Tip:' ),
+		(
+			<span className="editor-post-publish-panel__link" key="label">
+				{ __( 'Add tags to your post' ) }
+			</span>
+		),
+	];
+
+	return (
+		<PanelBody initialOpen={ false } title={ panelBodyTitle }>
+			<p>
+				{ __( 'Enter a few words that describe your post’s subject. This will help interested readers find it.' ) }
```

Comment:
Also: where/what readers will this help? Do we mean on WordPress.com? I think this lacks context and I wouldn't know what it would mean when presented with it.

Reply:
e150fbc

Reply:
I'm OK with this but I'll defer to what Sarah/Michelle think here: https://github.com/WordPress/gutenberg/pull/8235#issuecomment-411387138

Reply:
This isn't for WordPress.com at all. Tags are good for SEO and to help users find related posts across your own site. [This](https://yoast.com/tagging-posts-properly-for-users-and-seo/) is a pretty good article if we want to add more context, but I'm wary about linking to anything from a UI that isn't coming directly from WordPress.

Some copyedit suggestions for this and other parts of this UI below, in the interests of keeping everything in one place.